source/RN/Kubernetes-Operator-for-PS-RN0.2.0.rst

```diff
@@ -7,52 +7,45 @@
 :Date: June 30, 2022
 :Installation: `Installing Percona Operator for MySQL <https://www.percona.com/doc/kubernetes-operator-for-mysql/ps/index.html#advanced-installation-guides>`_
 
-.. note:: Version 0.2.0 of the Percona Operator for MySQL is **a tech preview release** and it is **not recommended for production environments**. **As of today, we recommend using** `Percona Operator for MySQL based on Percona XtraDB Cluster <https://www.percona.com/doc/kubernetes-operator-for-pxc/index.html>`_, which is production-ready and contains everything you need to quickly and consistently deploy and scale MySQL instances in a Kubernetes-based environment on-premises or in the cloud.
+.. note:: Version 0.2.0 of the Percona Operator for MySQL is **a tech preview release** and it is **not recommended for production environments**. **As of today, we recommend using** `Percona Operator for MySQL based on Percona XtraDB Cluster <https://www.percona.com/doc/kubernetes-operator-for-pxc/index.html>`_, which is production-ready and contains everything you need to quickly and consistently deploy and scale MySQL clusters in a Kubernetes-based environment, on-premises or in the cloud.
 
 Release Highlights
 ================================================================================
 
 * With this release, the Operator turns to a simplified naming convention and
   changes its official name to **Percona Operator for MySQL**
+* This release brings initial :ref:`implementation of Group Replication<mysql-clustertype>` between Percona Server for MySQL instances. Group Replication works in conjunction with MySQL Router, which is used instead of Orchestrator and also provides load balancing
+* Now the Operator :ref:`is capable of making backups<backups>`. Backups are stored on the cloud outside the Kubernetes cluster: `Amazon S3, or S3-compatible storage <https://en.wikipedia.org/wiki/Amazon_S3#S3_API_and_competing_services>`_ is supported, as well as `Azure Blob Storage <https://azure.microsoft.com/en-us/services/storage/blobs>`_. Currently, backups are work with asynchronous replication; support for backups with Group Replication is coming
```

Comment:
`Currently, backups are work with asynchronous replication; support for backups with Group Replication is coming` -> `Currently, backups work with asynchronous replication; support for backups with Group Replication is coming`